src/Components/RemediationsPage/RemediationsTable.tsx

```diff
@@ -0,0 +1,142 @@
+  const [activeSortIndex, setActiveSortIndex] = React.useState<number | null>(null);
+  const [activeSortDirection, setActiveSortDirection] = React.useState<'asc' | 'desc' | null>(null);
+  const [remmediations, setRemediations] = React.useState<Remediation[]>([]);
+
+  const { getRegistry } = useContext(RegistryContext);
+
+  const buildSort = (direction, index): string => {
+    if (direction === null || index === null) {
+      return 'updated_at';
+    }
+    const sortMap = {
+      0: 'name',
+      1: 'updated_at',
+      2: 'system_count',
+    };
+    const sign = direction === 'asc' ? '' : '-';
+    const sort = sign + sortMap[index];
+    return sort;
+  };
+
+  const fetchData = async (page: number, perPage: number, sort) => {
+    setShowSkeleton(true);
+
+    const response = await remediationsFetch(page, perPage, sort);
+    if (isError(response)) {
+      const store = getRegistry().getStore();
+      displayErrorNotification(store, response.message);
+      return;
+    }
+
+    const remediations = response.data.map((rem) => ({
+      id: rem.id,
+      name: rem.name,
+      updated_at: rem.updated_at,
+      system_count: rem.system_count,
+      resolved_count: rem.resolved_count,
+    }));
+
+    setRemediations(remediations);
+    const count = response.meta.count;
+    setTotal(count);
+
+    setShowSkeleton(false);
+    return count;
+  };
+
+  const getSortParams = (columnIndex: number): ThProps['sort'] => {
+    return {
+      sortBy: {
+        index: activeSortIndex as number,
+        direction: activeSortDirection as 'asc' | 'desc',
+        defaultDirection: 'asc',
+      },
+      onSort: (_event, index, direction) => {
+        setActiveSortIndex(index);
+        setActiveSortDirection(direction);
+      },
+      columnIndex,
+    };
+  };
+
+  useEffect(() => {
+    fetchData(page, perPage, buildSort(activeSortDirection, activeSortIndex));
+  }, [activeSortDirection, activeSortIndex, page, perPage]);
+
+  return (
+    <React.Fragment>
+      <Table aria-label="Recommendations" ouiaId="RecommendationsTable">
+        <Thead>
+          <Tr>
+            <Th sort={getSortParams(0)}>{columnNames.description}</Th>
+            <Th sort={getSortParams(1)}>{columnNames.updated_at}</Th>
+            <Th>{columnNames.status}</Th>
+            <Th sort={getSortParams(2)}>{columnNames.system_count}</Th>
+          </Tr>
+        </Thead>
+        {showSkeleton ? (
+          loadingSkeletons(perPage, Object.keys(columnNames).length)
+        ) : (
+          <Tbody>
+            {remmediations.length === 0 ? (
+              <Tr>
+                <Td colSpan={Object.keys(columnNames).length}>
+                  <Bullseye>
+                    <EmptyState variant={EmptyStateVariant.sm}>
+                      <EmptyStateHeader icon={<EmptyStateIcon icon={SearchIcon} />} titleText="No remediations found" headingLevel="h2" />
+                    </EmptyState>
+                  </Bullseye>
```

Comment:
Little bit prefer what actual remediations have, it's more informative

![image](https://github.com/RedHatInsights/in-place-upgrades-frontend/assets/19702477/3e12ae8d-a24f-4848-ba5a-3d7d2d55fefe)